apps/pie-docs/src/components/toast/toast-provider/code.md

```diff
@@ -0,0 +1,139 @@
+
+The `pie-toast-provider` component provides centralized management for toast notifications in an application. It ensures that toasts are displayed in [priority order](/components/toast/#priority-queue) based on their variant and actionable state, using a queueing system. The component handles the lifecycle of each toast, including dismissing the current toast and showing the next one.
+
+{% notification {
+  type: "information",
+  iconName: "engineers",
+  message: "You can try out this component on our [Storybook](https://webc.pie.design/?path=/docs/toast-provider) instance!"
+} %}
+
+## Installation
+
+To install `pie-toast-provider` in your application, run the following on your command line:
+
+```shell
+npm i @justeattakeaway/pie-webc
+```
+
+```shell
+yarn add @justeattakeaway/pie-webc
+```
+
+{% notification {
+  type: "neutral",
+  iconName: "link",
+  message: "For more information on using PIE components as part of an application, check out the [Getting Started Guide](https://github.com/justeattakeaway/pie/wiki/Getting-started-with-PIE-Web-Components)."
+} %}
+
+## Props
+
+{% componentDetailsTable {
+  tableData: props
+} %}
+
+## Events
+
+{% componentDetailsTable {
+  tableData: events
+} %}
+
+## Examples
+
+The usage guideline is:
+
+- Place `pie-toast-provider` at the root level of your application or page.
+- Use the `toaster` utility from any where in your app to dynamically create toasts.
+
+### Importing the component
+
+For HTML:
+
+```js
+// import as module into a js file e.g. main.js
+import '@justeattakeaway/pie-webc/components/toast-provider.js'
+```
+
+```html
+<!-- pass js file into <script> tag -->
+<pie-toast-provider></pie-toast-provider>
+<script type="module" src="/main.js"></script>
+```
+
+For Native JS Applications, Vue, Angular, Svelte etc.:
+
+```js
+// Vue templates (using Nuxt 3)
+import '@justeattakeaway/pie-webc/components/toast-provider.js';
+
+<pie-toast-provider></pie-toast-provider>
+```
+
+For React Applications:
+
+```jsx
+import { PieToastProvider } from '@justeattakeaway/pie-webc/react/toast-provider.js';
+
+<PieToastProvider></PieToastProvider>
+```
+
+### Creating Toasts with `toaster`
+
+The `toaster` utility dynamically creates toasts. It can be imported and called from any file or component in your application.
+
+```js
+import { toaster } from '@justeattakeaway/pie-webc/components/toast-provider.js';
+
+toaster.createToast({
+  message: 'This is a success toast!',
+  variant: 'success',
+  isDismissible: true,
+});
+
+```
+
+To clear all active and queued toasts:
+
+```js
+toaster.clearToasts();
+
+```
+
+{% notification {
+  type: "neutral",
+  iconName: "link",
+  message: "For more examples, see [here](https://github.com/justeattakeaway/pie-aperture/tree/main)."
+} %}
+
+## Changelog
+
+{% notification {
+  type: "neutral",
+  iconName: "link",
+  message: "See [here](https://github.com/justeattakeaway/pie/blob/main/packages/components/pie-toast/CHANGELOG.md)."
```

Comment:
```suggestion
  message: "See [here](https://github.com/justeattakeaway/pie/blob/main/packages/components/pie-toast-provider/CHANGELOG.md)."
```

Note this this URL doesn't currently exist as the component hasn't been published to npm yet, but the changelog.md will get generated in this location once that's been done 👍🏼